src/main/java/net/wurstclient/altmanager/AuthenticationException.java

```diff
@@ -0,0 +1,7 @@
+package net.wurstclient.altmanager;
+
+public class AuthenticationException extends Throwable {

```

Comment:
Is there any special reason for creating a separate `AuthenticationException` instead of using the existing `LoginException`? If not, please remove it.